lib/modules/model/types/ModelContent.ts

```diff
@@ -38,6 +38,7 @@ export interface MetadataDetails {
     locales: {
       [locale: string]: LocaleDetails;
     };
+    readOnly?: boolean;
```

Comment:
Instead of having this field optional, wouldn't it be more appropriate to have a sensible default value for this?

Reply:
Yes I have set the default value to false in the frontend if the field does not exist. If the field is not optional, there will be the need to update the metadataDetails mapping for every existing asset/device.